specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/examples/GetAGalleryImageVersion.json

```diff
@@ -0,0 +1,76 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2019-03-01",
+    "galleryName": "myGalleryName",
+    "galleryImageName": "myGalleryImageName",
+    "galleryImageVersionName": "1.0.0",
+    "galleryImageVersion": {
+      "location": "West US",
+      "properties": {
+        "publishingProfile": {
+          "targetRegions": [
+            {
+              "name": "West US",
+              "regionalReplicaCount": 1
+            },
+            {
+              "name": "East US",
+              "regionalReplicaCount": 2
+            }
+          ]
+        }
+      },
+      "storageProfile": {
+        "source": {
+          "id": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Compute/images/{imageName}"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "publishingProfile": {
+            "targetRegions": [
+              {
+                "name": "West US",
+                "regionalReplicaCount": 1,
+                "storageAccountType": "Standard_LRS"
+              },
+              {
+                "name": "East US",
+                "regionalReplicaCount": 2,
+                "storageAccountType": "Standard_LRS"
+              }
+            ],
+            "replicaCount": 1,
```

Comment:
can you give a GET example when source is from snapshots?

Reply:
added